src/style.css

```
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-basis: 33.33%;
+  flex-grow: 1;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  min-height: 300px;
+  min-width: 300px;
+  max-width: 640px;
```

use flex-basis for setting width
```suggestion
  height: 300px;
```